lib/Sema/CSBindings.cpp

```diff
@@ -1567,18 +1567,27 @@ void PotentialBindings::retract(Constraint *constraint) {
   EquivalentTo.remove_if(hasMatchingSource);
 }
 
-LiteralBindingKind BindingSet::getLiteralKind() const {
-  LiteralBindingKind kind = LiteralBindingKind::None;
-
+SmallVector<KnownProtocolKind> BindingSet::getProtocolKind() const {
```

Comment:
Not sure, but I guess naming here could be something that suggest plural? e.g. `getAllProtocolKinds()`(not very good at naming)?